### ISSUE TITLE
TypeError: Cannot read property 'client' of undefined in version 2.1.1

### ISSUE THREAD

**Issue by chgl** [smart-on-fhir/client-js#68]
Upgrading from v2.1.0 to 2.1.1 causes the error `TypeError: Cannot read property 'client' of undefined` when using `FHIR.client` directly.

Not sure if it makes a difference, but FHIR is used within a VueJS app (v2.6.11), e.g.:
```js
<template>
  <div id="app">
  </div>
</template>

<script>
import FHIR from "fhirclient";

export default {
  name: "App",
  components: {
  },
  async mounted() {
    const client = FHIR.client("http://hapi.fhir.org/baseR4");
    const bundle = await client.request("Patient");
    console.log(bundle);
  }
};
</script>
```

```
vue.runtime.esm.js?2b0e:1888 TypeError: Cannot read property 'client' of undefined
    at VueComponent.mounted (App.vue?234e:18)
    at invokeWithErrorHandling (vue.runtime.esm.js?2b0e:1854)
    at callHook (vue.runtime.esm.js?2b0e:4219)
    at Object.insert (vue.runtime.esm.js?2b0e:3139)
    at invokeInsertHook (vue.runtime.esm.js?2b0e:6346)
    at Vue.patch [as __patch__] (vue.runtime.esm.js?2b0e:6565)
    at Vue._update (vue.runtime.esm.js?2b0e:3945)
    at Vue.updateComponent (vue.runtime.esm.js?2b0e:4066)
    at Watcher.get (vue.runtime.esm.js?2b0e:4479)
    at new Watcher (vue.runtime.esm.js?2b0e:4468)
```

**Comment by vlad-ignatov** [comment]
This should be fixed now in `2.2.0`. I hope you can verify that it works for you.

**Comment by chgl** [comment]
Works perfectly, thank you!
